packages/vue/src/histoire-theme.css

```
@@ -0,0 +1,3 @@
+@import "tailwindcss";
+@plugin "daisyui";
+@import "@artisanpack-ui/tokens/css";
```

Stylelint (17.6.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)